src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -194,7 +194,12 @@ To avoid confusion, the channels SHOULD be listed in the order they
 appear in the EEG data file.
 Any number of additional columns may be added to provide additional information
 about the channels.
+
 Note that electrode positions SHOULD NOT be added to this file, but to [`*_electrodes.tsv`](./03-electroencephalography.md#electrodes-description-_electrodestsv).
+Furthermore, the entried in `*_electrodes.tsv` and `*_channels.tsv` do not have to match exactly,
+as for example in the case of recording a single `EOG` channel from a bipolar referencing scheme
```

Comment:
Would it be helpful to be explicit about what an example of this might look like? E.g. as you describe in https://github.com/mne-tools/mne-bids/issues/485

Reply:
I agree that that would be good for clarification. However putting the example right into this section breaks the flow a bit IMHO, because channels-tsv and electrodes-tsv and their examples are only introduced further below. Do you have a suggestion how to solve this?

Reply:
I think one way would be to adjust the two examples below (channels and electrodes) to refer to one another, instead of being two separate examples with no relationship.

And then to link to these two examples from this presently edited paragraph.

Reply:
Sounds good to me because `channels/electrodes` are inherently linked and dependent on each other.

Reply:
okay, I took a stab at this in https://github.com/bids-standard/bids-specification/pull/734/commits/a28c8b37f7c8c5d9f367b5c92ca46918f21ce668 - please review :)

Reply:
LGTM